hibernate-core/src/main/java/org/hibernate/dialect/Dialect.java

```diff
@@ -1420,8 +1435,14 @@ public String toString() {
 
 	// database type mapping support ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+
+	@Override
+	public void contribute(TypeContributions typeContributions, ServiceRegistry serviceRegistry) {
```

Comment:
I would prefer marking this as `final` so that people only have to override a the `contributeTypes` method in a custom dialect if necessary.

Reply:
Yeah sure, that's a good idea.